### PR TITLE
Decouple Homepage Hero Image Text/Rotator

### PR DESCRIPTION
**Pull request by maxxcrawford**
The homepage rotator now has the strings that were embedded in them outside, so that they can display in the users' requested language.

### CODE REVIEW

File: static/scss/pages/home.scss [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        @include text-title-3xs;
+        color: $color-purple-90;
+        text-align: center;
+        margin-bottom: $spacing-md;
+    }
+
+    img {
+        display: block;
+        width: $content-xs;
+        margin-left: auto;
```
Comment (codemist) [inline static/scss/pages/home.scss:57]: nit: `margin: 0 auto;`?

**Review by codemist — APPROVED** [review]
That's some Protocol-ed CSS!

File: static/scss/pages/home.scss (hunk 1 above)
Comment (maxxcrawford) [reply static/scss/pages/home.scss]: Revised. I had an additional commit that retooled a ton of the "lines" of the site with the `$content-max` width set everything. I ended up bailing on it, but I was constantly only using `margin-left/right` to preserve the Y axis margins from a parent style. 

TLDR; Great catch!
